modules/bridge/src/main/ts/ephox/bridge/components/dialog/ColorInput.ts

```diff
@@ -1,17 +1,21 @@
-import { StructureSchema, ValueType } from '@ephox/boulder';
+import { FieldSchema, StructureSchema, ValueType } from '@ephox/boulder';
 import { Result } from '@ephox/katamari';
 
 import { FormComponentWithLabel, formComponentWithLabelFields, FormComponentWithLabelSpec } from './FormComponent';
 
 export interface ColorInputSpec extends FormComponentWithLabelSpec {
   type: 'colorinput';
+  storageKey?: string;
```

Comment:
This needs a bridge changelog entry as well.

- [ ] Add a bridge changelog entry